index.bs

```diff
@@ -1610,33 +1609,71 @@ To <dfn>serialize an [=event-level report=]</dfn> |report|, run the following st
     [=serialize an integer|serialized=].
 1. Return the [=byte sequence=] resulting from executing [=serialize an infra value to JSON bytes=] on |data|.
 
+To <dfn>serialize an [=aggregatable report=]</dfn> |report|, run the following steps:
+
+1. Let |data| be a [=map=] of the following key/value pairs:
+
+    : "`shared_info`"
+    :: ""
+    : "`aggregation_service_payloads`"
+    :: ""
+
+1. If |report|'s [=aggregatable report/source debug key=] is not null, [=map/set=]
+    |data|["`source_debug_key`"] to |report|'s [=aggregatable report/source debug key=],
+    [=serialize an integer|serialized=].
+1. If |report|'s [=aggregatable report/trigger debug key=] is not null, [=map/set=]
+    |data|["`trigger_debug_key`"] to |report|'s [=aggregatable report/trigger debug key=],
+    [=serialize an integer|serialized=].
+1. Return the [=byte sequence=] resulting from executing [=serialize an infra value to JSON bytes=] on |data|.
+
+Issue: Spec |data|["`shared_info`"] and |data|["`aggregation_service_payloads`"].
+
+To <dfn>serialize an [=attribution report=]</dfn> |report|, run the following steps:
+
+1. [=Assert=]: |report|'s [=attribution report/attribution destination=] is not an [=opaque origin=].
+1. If |report| is an:
+    <dl class="switch">
+    <dt>[=event-level report=]</dt>
+    <dd>Return the result of running [=serialize an event-level report=] with |report|.</dd>
+
+    <dt>[=aggregatable report=]</dt>
+    <dd>Return the result of running [=serialize an aggregatable report=] with |report|.</dd>
+    </dl>
+
 Note: The inclusion of "`report_id`" in the report body is intended to allow the report recipient
```

Comment:
Is it possible to keep this note immediately below the `serialize an event-level report` dfn?

Reply:
This note may apply to aggregatable reports as well to avoid double counting?